### PR TITLE
Add Health/Armor saving

### PR DESCRIPTION
**Pull request by Holidayy95**
Goes along with PRs

https://github.com/qbcore-framework/qb-core/pull/678

https://github.com/qbcore-framework/qb-core/pull/679

### CODE REVIEW

File: client/job.lua [hunk 1]
```diff
@@ -122,6 +123,7 @@ RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
         QBCore.Functions.GetPlayerData(function(PlayerData)
             PlayerJob = PlayerData.job
             onDuty = PlayerData.job.onduty
+            SetEntityHealth(PlayerPedId(), PlayerData.metadata["health"])
             SetPedArmour(PlayerPedId(), PlayerData.metadata["armor"])
             if (not PlayerData.metadata["inlaststand"] and PlayerData.metadata["isdead"]) then
```
Comment (BerkieBb) [inline client/job.lua:128]: You can use `ped` here instead of `PlayerPedId()`

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client/job.lua (hunk 1 above)
Comment (Holidayy95) [reply client/job.lua:128]: Does it matter, or is it just normal practice?

Comment (BerkieBb) [reply client/job.lua:128]: No need to re-call that function when it's already assigned to a variable

Comment (Holidayy95) [reply client/job.lua:128]: yeah I see what you mean, it's already localised at the top of the function.

**Review by BerkieBb — APPROVED** [review]

**Comment by jnccloud** [comment]
I added one more check to this, and sent a PR to your patch @Holidayy95 

https://github.com/Holidayy95/qb-ambulancejob/pull/1

It should guarantee armor is always set to 0 on a revive incase the database is a little behind since last sync. This seemed to be the only area I ran into issues with while my solo testing, and resolved them.

**Pull request closed without merging**